adis/adis.py

```diff
@@ -158,3 +166,22 @@ def dumps(self):
 
     def __repr__(self):
         return """Adis containing %d Adis-files""" % len(self.files)
+
+    @staticmethod
+    def add_string_value(obj: list, entity_data_dict: dict):
+        """ Adds string value to definition list
+
+        Args:
+            obj (list): containing the logical adis files and their contents as builtin types without string values
+            entity_data_dict (dict): dictionary of mapping values for \
+             entity numbers (e.g. {"0080004": "Betriebsnummer"})
```

Comment:
if i get it right, those are item numbers and not entity numbers.
for example: the item 00800004 is included in the entity 870204 (http://ian.lkv-nrw.de/index.php?id=292&vURL=AGRO2018%2Fe870204.html&cHash=89029bb2ac)